docs/release-notes/7.0.md

```diff
@@ -0,0 +1,228 @@
+# Release notes for Gluster 7.0
```

Comment:
I already see release notes for 7 live on the site https://docs.gluster.org/en/latest/release-notes/7.0/ is this a inadvertent repeat?

Reply:
I did not understand this. Isn't release notes suppose to be in gluster docs also ?
So it is there at two places : 
1. In gluster docs 
2. In glusterfs code base